requirements/base.txt

```diff
@@ -1,31 +1,23 @@
-# Include the requirements we're caching as "wheel" archives
-# to speed up the test builds.
--r wheels.txt
-
 # edX Internal Requirements
-git+https://github.com/edx/XBlock.git@xblock-0.4.12#egg=XBlock==0.4.12
+# edx-submissions>=2.0.6,<3.0.0 #TODO: update after 2.0.6 release
+git+https://github.com/edx/edx-submissions.git@efischer/loosen_reqs#egg=edx-submissions==2.0.6 # TODO: and remove this line
```

Comment:
These two lines are effectively "I'm waiting for https://github.com/edx/edx-submissions/pull/65 to merge before this"